### PR TITLE
Fix error when inviting already invited user

### LINKED ISSUE #2421: Trying to send invite to already invited email throws server error
### Expected behavior
If you try to invite a user you have already invited (as determined by email address) it should give you a friendly validation-type error

### Actual Behavior
Receive a toast of "Server Error [500]" on the client and this error on the server:
```
Exception while invoking method 'accounts/inviteShopMember' ReferenceError: subject is not defined
    at [object Object]._inviteShopMember (server/methods/accounts/accounts.js:553:7)
    at packages/check.js:129:16
    at [object Object]._.extend.withValue (packages/meteor.js:1122:17)
    at Object.exports.Match._failIfArgumentsAreNotAllChecked (packages/check.js:128:41)
    at maybeAuditArgumentChecks (packages/ddp-server/livedata_server.js:1734:18)
    at packages/ddp-server/livedata_server.js:719:19
    at [object Object]._.extend.withValue (packages/meteor.js:1122:17)
    at packages/ddp-server/livedata_server.js:717:40
    at [object Object]._.extend.withValue (packages/meteor.js:1122:17)
    at packages/ddp-server/livedata_server.js:715:46
    at [object Object]._.extend.protocol_handlers.method (packages/ddp-server/livedata_server.js:689:23)
    at packages/ddp-server/livedata_server.js:559:43
```

### Steps to Reproduce the Behavior
1. Login as admin
1. Invite a user
1. Attempt to invite a user with that same email address
1. Observer client/server error

### Versions
```
Node: 7.10.0
NPM: 4.2.0
Meteor Node: 4.8.3
Meteor NPM: 4.6.1
Reaction CLI: 0.9.2
Reaction: 1.2.0
Reaction branch: development
Docker: 17.03.1-ce
```

### PR DESCRIPTION
**Pull request by awadesmoline**
Resolves #2421 

### Steps to Test the Fix
1. Login as admin
1. Invite a user
1. Attempt to invite a user with that same email address
1. Observer client/server response

### CODE REVIEW

**Comment by zenweasel** [comment]
@awadesmoline So you will want to give the PR a more meaningful title as this is what will show up in the commit history when the PR is merged.

**Comment by zenweasel** [comment]
This error message looks funky.

reaction

**Comment by awadesmoline** [comment]
@zenweasel Okay.

**Comment by awadesmoline** [comment]
@zenweasel Hope this is okay?
screen shot 2017-06-14 at 3 03 59 pm

**Comment by zenweasel** [comment]
@rymorgan How do we want to do an error message like this? Is a toast okay?

**Comment by rymorgan** [comment]
@zenweasel it should be an inline alert instead. The message also needs to be more useful and specific. “A user with this email address already exists”

File: server/methods/accounts/accounts.js [hunk 1]
```diff
@@ -499,84 +557,21 @@ export function inviteShopMember(shopId, email, name) {
...
       html: SSR.render(tpl, dataForEmail)
     });
   } else {
-    Reaction.Email.send({
-      to: email,
-      from: `${shop.name} <${shop.emails[0].address}>`,
-      subject: SSR.render(subject, dataForEmail),
-      html: SSR.render(tpl, dataForEmail)
-    });
+    throw new Meteor.Error(409, "User already exists");
```
Comment (zenweasel) [inline server/methods/accounts/accounts.js]: Needs i18n

**Comment by zenweasel** [comment]
@rymorgan What should the inline alert look like?

**Comment by rymorgan** [comment]
@zenweasel Yes sir.

**Comment by rymorgan** [comment]
@zenweasel -- Should have added this yesterday. Here's some additional guidance on alerts in general and where I want us to go. 

**Alerts**
https://zpl.io/1GI9nB

**Comment by zenweasel** [comment]
Getting this error in the console when I test:
```
meteor.js?hash=27829e9…:930 Exception in delivering result of invoking 'accounts/inviteShopMember': TypeError: Cannot read property 'set' of undefined
    at http://localhost:3000/app/app.js?hash=be086c2daed8d005c68de6276530772ad6166fbe:86587:32
    at MethodInvoker._callback (http://localhost:3000/packages/meteor.js?hash=27829e936d09beae3149ecfbf3332c42ccb1596f:1105:22)
    at MethodInvoker._maybeInvokeCallback (http://localhost:3000/packages/ddp-client.js?hash=c9ca22092a3137a7096e8ab722ba5ab8eedb9aec:3678:12)
    at MethodInvoker.receiveResult (http://localhost:3000/packages/ddp-client.js?hash=c9ca22092a3137a7096e8ab722ba5ab8eedb9aec:3698:10)
    at Connection._livedata_result (http://localhost:3000/packages/ddp-client.js?hash=c9ca22092a3137a7096e8ab722ba5ab8eedb9aec:4812:9)
    at onMessage (http://localhost:3000/packages/ddp-client.js?hash=c9ca22092a3137a7096e8ab722ba5ab8eedb9aec:3525:206)
    at http://localhost:3000/packages/ddp-client.js?hash=c9ca22092a3137a7096e8ab722ba5ab8eedb9aec:2908:9
    at Array.forEach (<anonymous>)
    at Function._.each._.forEach (http://localhost:3000/packages/underscore.js?hash=cde485f60699ff9aced3305f70189e39c665183c:149:11)
    at SockJS.self.socket.onmessage (http://localhost:3000/packages/ddp-client.js?hash=c9ca22092a3137a7096e8ab722ba5ab8eedb9aec:2907:43)
```

**Comment by awadesmoline** [comment]
@zenweasel, error has been fixed.

**Pull request merged**